components/arm.py

```diff
@@ -180,7 +181,7 @@ def __init__(self) -> None:
         wpilib.SmartDashboard.putData("Arm sim", self.arm_mech2d)
 
     def setup(self) -> None:
-        self.set_setpoint(Setpoints.SCORE_CONE_HIGH)
+        self.goto_setpoint(Setpoints.SCORE_CONE_HIGH)
```

Comment:
Is this the starting angle we want? I would think we want to setup in stow or whatever our natural position is and then command the intake or score high when its appropriate.

```suggestion
        self.goto_setpoint(Setpoints.STOW)
```